indico/modules/events/abstracts/blueprint.py

```diff
@@ -48,6 +48,8 @@
 
 # Misc
 _bp.add_url_rule('/abstracts/other-list', 'other_abstracts', reviewing.RHListOtherAbstracts, methods=('POST',))
+_bp.add_url_rule('/abstracts/invited/<int:abstract_id>', 'submit_invited_abstract', display.RHSubmitInvitedAbstract,
```

Comment:
should we use the UUID here instead of the abstract id?

Reply:
Hm, I don't see any reason why. It would for sure improve the readability of the links we send to invited submitters.